frontend/src/components/FAQs.tsx

```diff
@@ -0,0 +1,342 @@
+import plusIcon from "../assets/plusIcon.svg";
+import { Accordion } from "../components/index.ts";
+import styles from "../stylesheets/Home.module.css";
```

Comment:
Looks like the styles you are pulling are only used in FAQs so lets move those styles into its own  styles into its own file in the same folder as FAQs page.